sdks/uniswapx-sdk/src/utils/order.ts

```diff
@@ -147,3 +158,19 @@ export class RelayOrderParser extends OrderParser {
     return RelayOrder.parse(order, chainId);
   }
 }
+
+export function isCosignedV2(
```

Comment:
WDYT about having both `UnsignedV2DutchOrder` and `UnsignedV3DutchOrder` implement an interface that has a function `isCosigned()`? Then we can run `order.isCosigned()` and it know how to perform the check.

Reply:
I see how we can do this more cleanly. I think in the future an interface would be useful if we do a larger refactoring of Dutch V2 & V3 to share more code together. For now since we're just interested in accessing cosigned status, I replaced the 2 functions with a single one that uses generics. Lmk what you think!

Reply:
Though of a clean way to just use `instanceof`!